.storybook/addon-vuetify/decorator.js

```diff
@@ -0,0 +1,42 @@
+// Imports
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import { makeDecorator } from '@storybook/addons'
+import { preset } from '../../src/preset'
+
+// Utilities
+import deepmerge from 'deepmerge'
+
+// Vuetify
+import 'vuetify/dist/vuetify.min.css'
```

Comment:
i wonder why we need compiled css and 
  transpileDependencies: [
    'vuetify'
at the same time.  Isnt we either use full bundle transpiled with css or transpile it with css too?